ide-migration/server/migration/process/list-available-databases-task.js

```diff
@@ -1,27 +1,27 @@
-const process = require('bpm/v4/process');
+const process = require("bpm/v4/process");
 const execution = process.getExecutionContext();
-const TrackService = require('ide-migration/server/migration/api/track-service');
+const TrackService = require("ide-migration/server/migration/api/track-service");
 const trackService = new TrackService();
 try {
```

Comment:
Using two spaces instead of two tabs(4 spaces) makes scope of try catches a bit more difficult to see at a glance.